front/src/common/redux/set-password/set.password.saga.js

```diff
@@ -0,0 +1,22 @@
+import { put, call, all, takeEvery } from 'redux-saga/effects';
+import { setPassswordApi } from './set.password.api';
+import * as types from './types';
+
+function* setPasswordSaga(payload) {
+    try {
+        const response = yield call(setPassswordApi, payload);
+        yield put({ type: types.SET_PASSWORD_SUCCESS, response });
+    } catch (error) {
+        yield put({ type: types.SET_PASSWORD_ERROR, error });
```

Comment:
Please, use in actions only `type` and `payload` inside of actions

Reply:
Fixed